lib/rust/profiler/flame-graph/src/lib.rs

```diff
@@ -6,15 +6,25 @@
 #![deny(non_ascii_idents)]
 #![warn(unsafe_code)]
 
+use crate::Kind::Active;
+use crate::Kind::Paused;
+
 use enso_profiler as profiler;
 use enso_profiler_data as data;
 
 
-
 // ==================
 // === Block Data ===
 // ==================
 
+/// Indicates whether a block indicates an active interval or a paused interval. Paused intervals
+/// are used to represent async tasks that are started and awaited, but that have made no progress.
+#[derive(Copy, Clone, Debug)]
+pub enum Kind {
```

Comment:
"Kind" is very general, maybe "State"?

Reply:
The design will actually be changed to accept a colour for the theme in the next PR after this, for more flexibility. Happy to change it to `State` but I would not spend much more time on coming up with a better name.